### ISSUE TITLE
Finish up with CBET Trip #3

### ISSUE THREAD

**Issue by ct-martin** [RITlug/config-management#10]
Schedule another trip to the CBET datacenter to try to finish setting up the servers.

**Comment by justwheel** [comment]
What is the next step to move this forward?

**Comment by ct-martin** [comment]
I have info now, need to email Dr. Skuse to see what his schedule looks like

**Comment by Tjzabel** [comment]
### Update
Dr. Skuse was unavailable during the time we were going to visit the CBET machines. 
We have since rescheduled, and are planning on sending him a secondary reminder a day or two before the meeting.

**Comment by justwheel** [comment]
This was completed this week. @ct-martin, is there anything remaining before this is closed out?

**Comment by jrtechs** [comment]
# Eboard meeting Notes

We got some stuff running and online from trip 3.

@ct-martin will write a follow up email with Dr. Skuse.

TLDR work on CBET trip # 4

**Comment by ct-martin** [comment]
@Tjzabel I need to document passwords, will look at in an hour

**Comment by ct-martin** [comment]
Passwords were added to creds (private) repo; closing this issue
